src/posts/interning-at-khan-academy.md

```diff
@@ -0,0 +1,53 @@
+title: "Interning at Khan Academy: from student to intern"
+published_on: November 19, 2016
+author: Shadaj Laddad
+team: Web Frontend
+...
+
+This past summer, I had the amazing opportunity to work on a product I use as a student every day -- Khan Academy. During my internship, I worked on the web frontend team on the XOM (x-on-mobile) initiative to bring Khan Academy exercises to mobile devices. In this blog post, I’ll be going through a highly-compressed summary of my two months at Khan Academy, but since most of my work here was in open-source projects you can check out my work in detail on GitHub ([https://github.com/Khan/perseus/commits?author=shadaj](https://github.com/Khan/perseus/commits?author=shadaj), [https://github.com/Khan/math-input/commits?author=shadaj](https://github.com/Khan/math-input/commits?author=shadaj)).
+
+# Part 1: ramping up
+
+My first day at Khan Academy was a whirlwind of activities. From setting up my laptop to having a 1:1 with my mentor and finally adding myself to the [interns page](https://www.khanacademy.org/about/our-interns), I had a lot of things to do. As soon as I settled down with my first deploy, I got to work on the XOM initiative, starting with improving the math keypad.
+
+As part of the XOM initiative, the team had designed a math keypad that could be used to type in expression responses to mathematics problems in a mobile-friendly way. I started by fixing some bugs to get familiar with the math keypad codebase, such as disabling gestures on the static sidebar, a fixed bar on the side of the keypad with navigation buttons. Once I had a good idea of how the keypad worked, I was able to start working on adding new features our designers had crafted. I worked on things such as using square icons for representing empty blocks, and learned in the process how important it is to make these small changes to create a great experience for students.
```

Comment:
Style nits:

* The phrase "the XOM initiative" is bulky, and it's used here in rapid succession. Shorten this instance to "As part of XOM".

Stray thoughts:

* It's probably okay to omit "to get familiar with the math keypad codebase", because the next sentence starts with an indication that you gained familiarity through the process and that the lack of familiarity had previously blocked feature work.
* The explanation of the static sidebar can probably be compressed a bit. Does "disabling gestures on the keypad's sidebar" feel sufficient to you? Might also be good to, if possible, briefly explain why the sidebar shouldn't respond to gestures (which I'm kinda wondering myself :D).
* Might be cute to have each bug/feature be a link to the most representative commit!
* I think the last couple sentences can be refactored a bit. If there's two more features worth naming, then this structure has pretty nice pacing: "Then, once I understood how the keypad worked, I was able to start implementing new features from the spec: X, Y, and Z, to name a few. And, by working on these low-level details, I learned the difference between a good user experience and a _great_ user experience: polish."